mmdet/models/dense_heads/embedding_rpn_head.py

```diff
@@ -101,6 +101,7 @@ def simple_test_rpn(self, img, img_metas):
         """Forward function in testing stage."""
         return self._decode_init_proposals(img, img_metas)
 
+    # TODO: is right ?
```

Comment:
Just delete it, `embedding_rpn_head` is not a `Detector` which should not have `show_result`.